Update the API "login" action documentation (#286)
kuzzleio/documentation: src/protocols/1/native-protocols/socketio/index.md

---
layout: full.html.hbs

> title: SocketIO

Author: ```suggestion
title: Socket.IO
```

order: 0
---

> # Socket IO

Author: ```suggestion
# Socket.IO
```

## Configuration

> The protocol can be configured via [Kuzzle'rc configuration]({{ site_base_path }}guide/1/essentials/configuration/), under ``server > protocols > socketio`` section.

Author: ```suggestion
The protocol can be configured via the [kuzzlerc configuration file]({{ site_base_path }}guide/1/essentials/configuration/), under the ``server > protocols > socketio`` section.
```

| Option | Type | Description | Default |
|---|---|---|---|

> | ``enabled`` | <pre>boolean</pre> | Enables/Disables SocketIO protocol support | ``true`` |

Author: ```suggestion
| ``enabled`` | <pre>boolean</pre> | Enable/Disable Socket.IO protocol support | ``true`` |
```

| ``origins`` | <pre>string</pre> | Value of Access-Control-Allow-Origin header to answer the upgrade request | ``*:*`` |

### Configure listening port

<div class="alert alert-warning">

> HTTP, WebSocket and Socket IO protocols share the same underlying server instance. Modifying the listening port will impact all these three protocols.

Author: ```suggestion
HTTP, WebSocket and Socket.IO protocols share the same underlying server instance. Modifying the listening port will impact all these three protocols.
```
